stdlib/src/collections/list.mojo

```diff
@@ -814,8 +814,10 @@ struct List[T: CollectionElement](CollectionElement, Sized, Boolable):
         debug_assert(
             0 <= idx < len(self[]),
             (
-                "The index provided must be within the range [0, len(List) -1]"
-                " when using List.unsafe_get()"
+                "The index provided must be within the range [0, "
+                + str(len(self[]) - 1)
```

Comment:
FYI I had to drop these internally since you can't use `str(...)` here as it will fail to interpret for cases where `List` is used at compile time since `str(...)` doesn't work at compile time which is a known issue.